ako-operator/config/crd/bases/ako.vmware.com_akoconfigs.yaml

```diff
@@ -91,6 +91,11 @@ spec:
                       labelValue:
                         type: string
                     type: object
+                  primaryInstance:
+                    description: PrimaryInstance defines AKO instance is primary or
```

Comment:
dont you have a default here ?

Reply:
For `boolean` fields, we are not specifying default.